README.md

```diff
@@ -760,8 +760,7 @@ This is a known issue:
 
 ### vim.fn.{function}()
 
-`vim.fn` is for calling a Vimscript function.
-The Vimscript function can either be a built-in Vim function or a user function. Data types are converted back and forth from Lua to Vimscript.
+`vim.fn` can be used to call a user Vimscript function. Data types are converted back and forth from Lua to Vimscript.
```

Comment:
```suggestion
`vim.fn` can be used to call a Vimscript function. Data types are converted back and forth from Lua to Vimscript.
```

You can also call builtin Vimscript functions like `has()`

Other than that, LGTM! Thanks again